### PR TITLE
Add translatable help page

### PR DESCRIPTION
**Pull request by danielfdsilva**

### CODE REVIEW

**Comment by olafveerman** [comment]
This looks good. I added a first draft for the Help page.

@danielfdsilva would it be possible make the anchors work? This will be important if we stick to a one-page help text.

File: app/content/help/en.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 Every project needs 5 different inputs to calculate accessibility for a region:
 
-1. [administrative boundaries](#administrative-boundaries)
-2. [population data](#population-data)
-3. [points of interest](#points-of-interest)
-4. [road network](#road-network)
-5. [OSRM profile](#profile)
+1. [administrative boundaries](#/en/help#administrative-boundaries)
+2. [population data](#/en/help#population-data)
+3. [points of interest](#/en/help#points-of-interest)
```
Comment (danielfdsilva) [inline app/content/help/en.md]: @olafveerman Anchor support was added. Links need the full path for it to work.

File: app/content/help/en.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+4. [road network](#/en/help#road-network)
```
Comment (danielfdsilva) [inline app/content/help/en.md]: @olafveerman Anchor support was added. Links need the full path for it to work.

**Comment by olafveerman** [comment]
Great, thanks!

**Pull request merged**